gigasecond/gigasecond.go

```diff
@@ -0,0 +1,17 @@
+// This is a "stub" file.  It's a little start on your solution.
+// It's not a complete solution though; you have to write some code.
+
+// Exercism package provinding a function adding a gigasecond to an
+// input date
+package gigasecond
+
+import "time"
+
+// Adds a gigasecond to an input time
+// parameters
+// * inputTime : time.Time structure to which this function will add a
+//				 Gigasecond
+func AddGigasecond(inputTime time.Time) time.Time {
+	gigasecondDuration := time.Duration(1000000000) * time.Second
```

Comment:
nit: `time.Duration(1000000000)` could be extracted in a constant `gigaSecond` maybe.

Reply:
Thanks ! Do we have any code practice regarding constants naming/case ? e.g. all uppercase as in the last push ?